index.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+entries:
+  sail-operator:
+  - apiVersion: v2
+    appVersion: 0.1.0
+    created: "2024-09-23T12:51:47.932616888+03:00"
+    description: A Helm chart for Kubernetes
+    digest: 94e0237a3476ef3f4784b299ec976bab05fdc73ccf4d978037ad996f0aa73c9f
```

Comment:
When I quickly verified the sha256sum of the [release tar](https://github.com/istio-ecosystem/sail-operator/releases/download/0.1.0/sail-operator-0.1.0.tgz), I'm seeing a different digest. Please check the digest value once.

Reply:
That is because I generated a new tar by using the `helm package ...` command and not took the artifact attached to the release.
I switched to the `release-0.1` branch.
I did it mostly because as part of the process I also created an `index.yaml` file which references the existing artifacts when the `helm repo add` command is used.

If you think I should rather add an artifact attached to the release, I could do it.

Reply:
Got it. I'd let @dgn to comment if we want to use release artifact or the latest release-0.1 branch.

Reply:
IMO it should be the same artifact from the Releases section, otherwise people might be confused/alarmed

Reply:
No problem.
Will modify it.

Reply:
Done